README.md

```diff
@@ -1,94 +1,187 @@
-# The ops library
+# The `ops` library
 
-<!-- The text below is also at the top of ops/__init__.py. Keep in sync! -->
 
-The ops library is a Python framework ([`available on PyPI`](https://pypi.org/project/ops/)) for developing
-and testing [Juju](https://juju.is/) charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
+The `ops` library is a Python framework for developing and testing charms in a consistent way, using standard Python constructs to allow for clean, maintainable, and reusable code. It is an official component of the Charm SDK, itself a crucial part of the Juju universe.
 
-A charm is an operator -- business logic encapsulated in a reusable software
-package that automates every aspect of an application's life.
+> ✔️ `ops` is  ([`available on PyPI`](https://pypi.org/project/ops/)).
+>
+> ⚠️ The latest version of `ops` requires Python 3.8 or above.
+> 
+> 🥇 While charms can be written in any language, `ops` defines the latest standard. All new charms are encouraged to use it.
```

Comment:
Call me a grumpy old man :-), but I don't think these particular emojis add any value. The check mark has little to do with PyPI, the warning sign isn't really a warning, and medal != standard. I think we could just make these bullet points. Suggestion (also note slight tweaks):

```suggestion
- `ops` is  [available on PyPI](https://pypi.org/project/ops/).
- The latest version of `ops` requires Python 3.8 or above.
- While charms can be written in any language, `ops` defines the latest standard. All new charms are encouraged to use Python with `ops`.
```

Reply:
I'll drop the emojis. I'll also try to move the third bullet in the introduction, as it seems to fit better there. (And some of the content there seems unnecessary.)

Reply:
Feel free to call me a grumpy old man too :joy:, but I agree about the emojis - we still have the right-hand-pointing one in the table at the top. I'm not sure why that row gets/needs an emoji?

I don't feel super strongly about it (and in all seriousness, maybe I am showing my age) so if it has value then ignore this comment.

Reply:
> we still have the right-hand-pointing one in the table at the top

I wanted the table to act like a map to the Juju universe. I'm using it for the Juju and Charmcraft READMEs too. The right-hand-pointing emoji is supposed to be the "You are here" mark. PS I'm hoping some day we can replace it with a nice visual to the same effect -- ideally a visual with clickable links, just like in the table.